crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -1194,7 +1194,7 @@ mod tests {
 
         // When I send sliding sync response containing an invited room
         let mut room = v4::SlidingSyncRoom::new();
-        set_room_invited(&mut room, user_id);
+        set_room_invited(&mut room, user_id, user_id);
```

Comment:
This look like a bit weird, `inviter` and `invitee` are distinct users, no? (repeated 3 times, line 1221 and 1244). So maybe create a distinct user?

Reply:
Oh this `set_room_invited` is a test function, and I've checked that all those tests don't actually care about the inviter/invitee distinction, since they observe other side-effects.

Reply:
OK, thanks for checking!